.github/workflows/lint.yml

```diff
@@ -6,8 +6,19 @@ on:
     branches: [master]
 
 jobs:
+  lint-c:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+    - name: install clang-format
+      run: sudo apt-get install clang-format clang-format-6
```

Comment:
We should just be able to delete ``clang-format-6`` and it'll work, right?

Reply:
You'd think so right! 
```
The following packages have unmet dependencies:
 clang-format : Depends: clang-format-6.0 (>= 6.0~) but it is not going to be installed
```

Reply:
I'm an idiot and didn't read the error correctly, unlike 7, 8 and 9 it is 6.0.

Reply:
Doesn't work with 6.0 either:
```
Te following packages have unmet dependencies:
 clang-format-6.0 : Depends: libllvm6.0 (= 1:6.0-1ubuntu2) but 1:6.0.1~svn334776-1~exp1~20190309042703.125 is to be installed
```
I'm firing up a "bionic minimal" VM to try.

Reply:
Lame! clang-format-7 is fine, it's just for running CI. One of us will spot it quickly if something gets added that doesn't work on 6.

Reply:
Fixed! The github image has a microsoft azure ppa that installs a cutom LLVM build.